presto-main/src/main/java/com/facebook/presto/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -1595,8 +1595,34 @@ public static ExpressionAnalyzer createWithoutSubqueries(
             String message,
             boolean isDescribe)
     {
-        return new ExpressionAnalyzer(functionRegistry, typeManager, node -> {
-            throw new SemanticException(errorCode, node, message);
-        }, session, ImmutableMap.of(), parameters, isDescribe);
+        return createWithoutSubqueries(
+                functionRegistry,
+                typeManager,
+                session,
+                ImmutableMap.of(),
```

Comment:
you are passing empty parameters from both places, maybe you could remove this parameter

Reply:
I am not sure I follow. What would you like to see removed?